### PR TITLE
FPGA interrupt APIs

### PR DESCRIPTION
**Pull request by amehra-ni**
### What does this Pull Request accomplish?

This PR contains code generation for Interrupt FPGA functions.

### Why should this Pull Request be merged?

- Updated metadata.
- Updated the custom service file by removing unnecessary code.

### What testing has been done?

- No compile or codegen issues.
- Manually verified server handlers for all APIs and compared them with [documentation for the APIs mentioned](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Fsrc%2Frio%2Friodriverd%2Ffpgainterfacecapi%2Fsource%2Fcvi%2Fbin%2FNiFpga_fp.h&version=GBmain&_a=contents).
- Manually tested these two APIs with client code which calls the functions and data comes as expected.

### CODE REVIEW

File: generated/nifpga/nifpga_library.cpp [hunk 1]
```diff
@@ -139,6 +141,14 @@ NiFpga_Status NiFpgaLibrary::Abort(NiFpga_Session session)
   return function_pointers_.Abort(session);
 }
 
+NiFpga_Status NiFpgaLibrary::AcknowledgeIrqs(NiFpga_Session session, uint32_t irqs)
+{
+  if (!function_pointers_.AcknowledgeIrqs) {
+    throw nidevice_grpc::LibraryLoadException("Could not find NiFpga_AcknowledgeIrqs.");
```
Comment (doshirohan) [inline generated/nifpga/nifpga_library.cpp:147]: I just noticed this, and this should not be fixed in this PR but should this read `Could not find NiFpgaDll_AcknowledgeIrqs`? You might need to fix something in codegen to fix this error.

Comment (doshirohan) [reply generated/nifpga/nifpga_library.cpp:147]: This is not a blocking comment for this review. Feel free to resolve this and merge the change, but do check if this suggestion makes sense.

File: generated/nifpga/nifpga_service.cpp [hunk 2]
```diff
@@ -63,6 +63,29 @@ namespace nifpga_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiFpgaService::AcknowledgeIrqs(::grpc::ServerContext* context, const AcknowledgeIrqsRequest* request, AcknowledgeIrqsResponse* response)
```
Comment (doshirohan) [inline generated/nifpga/nifpga_service.cpp:68]: Did you try to call these 2 APIs over gRPC and verify they work as expected? I didn't see anything about this in your review description.

**Review by doshirohan — APPROVED** [review]

**Review by astarche — APPROVED** [review]

**Review by reckenro — APPROVED** [review]

File: generated/nifpga/nifpga_library.cpp (hunk 1 above)
Comment (amehra-ni) [reply generated/nifpga/nifpga_library.cpp:147]: Sure, I'll take care of this in follow-up

**Pull request merged**